event/consumer.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	kafka "github.com/ONSdigital/dp-kafka/v2"
+	dpnetrequest "github.com/ONSdigital/dp-net/request"
```

Comment:
Can we alias this import to `dprequest`? That way it aligns with how we alias other imports which conflict with Gos inbuilt libraries